types/pg/index.d.mts

```diff
@@ -0,0 +1,21 @@
+import type * as pg from "./index.d.ts";
+
+import TypeOverrides = require("./lib/type-overrides");
+
+export type * from "./index.d.ts";
```

Comment:
I don't think a type reexport is enough. In the original code, all properties of the cjs entry point are also reexported in the esm entry point: https://app.unpkg.com/pg@8.15.6/files/esm/index.mjs#L4-14

Reply:
The thing is this is a type-only export, but we also want to export values defined in `index.d.ts` (e.g. `Client`).

Reply:
ok I see what you mean now ^^;;
fixed!